web/components/njwds-extended/VerticalStepIndicator.tsx

```diff
@@ -3,10 +3,12 @@ import { useMountEffect, useOnWindowResize } from "@/lib/utils/helpers";
 
 interface Props {
   number: number;
+  order?: number;
```

Comment:
I see (now) why this prop is needed re the discussion about resetting step numbering for each section. But I think the name is confusing, this doesn't change anything for the step ordering? Instead it seems to represent the way we display the step number to the user. Can we rename the prop for clarity?

Reply:
Also, why optional?